instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go

```diff
@@ -54,9 +54,6 @@ func NewServerHandler(opts ...Option) stats.Handler {
 
 // TagConn can attach some information to the given context.
 func (h *serverHandler) TagConn(ctx context.Context, info *stats.ConnTagInfo) context.Context {
-	span := trace.SpanFromContext(ctx)
-	attrs := peerAttr(peerFromCtx(ctx))
-	span.SetAttributes(attrs...)
 	return ctx
 }
```

Comment:
Why is this method kept at all? It is not doing anything?

Was this originally used by users to add attributes to their spans?

Reply:
This method is required by the interface: https://pkg.go.dev/google.golang.org/grpc/stats#Handler

> Was this originally used by users to add attributes to their spans?

No. It was never working.